static/js/ui_init.js

```diff
@@ -39,7 +39,7 @@ function message_hover(message_row) {
         return;
     }
 
-    // But the message edit hover icon is determined by whether the message is still editablex
+    // But the message edit hover icon is determined by whether the message is still editable
```

Comment:
This doesn't have to do with my code; just found a small typo

Reply:
Well spotted! Would you send this as a separate commit?

For splitting up a commit into separate pieces, `git add -p` is a handy tool.